<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Solithium Power</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pb_SunSolar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>94</y>
      <width>114</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/SunSolarPanel.jpg</normaloff>:/Icons/SunSolarPanel.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>105</width>
      <height>105</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ChargeController">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>291</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/ChargeController.png</normaloff>:/Icons/ChargeController.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>105</width>
      <height>105</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_SolarDnArrow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>194</y>
      <width>31</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/Down Arrow.PNG</normaloff>:/Icons/Down Arrow.PNG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>62</width>
      <height>111</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ChargeControler_Arrow">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/ChargeController_LeftArrow.PNG</normaloff>:/Icons/ChargeController_LeftArrow.PNG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>129</width>
      <height>129</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Battery_Arrow">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>21</width>
      <height>129</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/Battery_Bidirectional_Arrow.PNG</normaloff>:/Icons/Battery_Bidirectional_Arrow.PNG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>129</width>
      <height>129</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BatteryPercentage">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>80</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>80%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Battery">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>92</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/Battery/Battery1.png</normaloff>:/Icons/Battery/Battery1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>118</width>
      <height>142</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BatteryVoltage">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>120</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>12.5 V</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ChargeControllerPower">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>300</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>+750 W</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_Power_Arrow">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/Power_RighArrow.PNG</normaloff>:/Icons/Power_RighArrow.PNG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>400</width>
      <height>129</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_External_Arrow">
    <property name="geometry">
     <rect>
      <x>483</x>
      <y>202</y>
      <width>21</width>
      <height>133</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/Battery_Bidirectional_Arrow.PNG</normaloff>:/Icons/Battery_Bidirectional_Arrow.PNG</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>129</width>
      <height>129</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_External">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>96</y>
      <width>84</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/RedBlackTerminals.jpg</normaloff>:/Icons/RedBlackTerminals.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>105</width>
      <height>105</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ExternalVoltage">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>14.4 V</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BatteryPower">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>+505 W</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ExternalPower">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>+406 W</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_LoadPower">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>290</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-1004 W</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_PowerButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>210</y>
      <width>76</width>
      <height>72</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/Off.png</normaloff>
      <normalon>:/Icons/On.png</normalon>
      <selectedon>:/Icons/Off.png</selectedon>:/Icons/Off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>62</width>
      <height>62</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_LightBulb">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>96</y>
      <width>84</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/Bulb_Off.png</normaloff>:/Icons/Bulb_Off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>105</width>
      <height>105</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SolarPanelCurrent">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>267</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5.3 A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SolarPanelVoltage">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>203</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>100 V</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ChargerVoltage">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>14.4 V</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ChargerCurrent">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>50.4 A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_LoadVoltage">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>330</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>14.4 V</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_LoadCurrent">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>50.4 A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_BatteryCurrent">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>50.4 A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ExternalCurrent">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>82</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>50.4 A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SolarPanelVoltage_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100 V</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SolarPanelVoltage_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>45</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100 V</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SolarPanelVoltage_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>45</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100 V</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SolarPanelVoltage_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>45</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100 V</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SolarPanelVoltage_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>45</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100 V</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SolarPanelVoltage_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>45</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100 V</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SolarPanelVoltage_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>45</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100 V</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SolarPanelVoltage_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>45</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100 V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_LeftArrow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>385</y>
      <width>76</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/ArrowL_SolarTracker.png</normaloff>:/Icons/ArrowL_SolarTracker.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>62</width>
      <height>61</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txt_CompLogo_">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>451</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Magneto</family>
      <pointsize>37</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>Solithium Power</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_RightArrow">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>385</y>
      <width>76</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="myResources.qrc">
      <normaloff>:/Icons/ArrowR_Config.png</normaloff>:/Icons/ArrowR_Config.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>62</width>
      <height>62</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txt_PowerOut_">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>POWER OUT</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_Inverter_">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inverter</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txt_BatteryChargers_">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>68</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Battery Chargers</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txt_DCDCconverters_">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>74</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DC/DC Converters</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txt_ExternalPower_">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>42</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>EXT. POWER</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_Terminals_">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Terminals</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txt_PowerStorage_">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>42</y>
      <width>161</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>POWER STORAGE</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_BatteryBank_">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>52</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Battery Bank</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txt_PowerIn_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>42</y>
      <width>65</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>POWER IN</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_SolarPanels_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Solar Panels</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="txt_SolarChargeController_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Charge Controller</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textLoggerBox">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>370</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myResources.qrc"/>
 </resources>
 <connections/>
</ui>
